soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.nrf52805_QFAA

```diff
@@ -0,0 +1,14 @@
+# Nordic Semiconductor nRF52805 MCU
```

Comment:
@anangl shouldn't this file be named with the right suffix, CAAA?

Reply:
Of course it should. Thanks for catching this!

Reply:
Fixed.